hw/ip/flash_ctrl/dv/flash_ctrl_sim_cfg.hjson

```diff
@@ -114,7 +114,12 @@
       run_opts: ["+zero_delays=1", "+test_timeout_ns=10000000000"]
       reseed: 3
     }
-
+    {
+      name: flash_ctrl_mp_regions
+      uvm_test_seq: flash_ctrl_mp_regions_vseq
+      run_opts: ["+test_timeout_ns=300000000000"]
```

Comment:
Thanks!

Reply:
Added.

Reply:
@weicaiyang 
@nikola-miladinovic can you clarify the cases where erase is taking such a long time? 
We did have an issue with erase that Weicai raised with the tool vendors, I just want to double check if we experiencing yet another round of this problem.

Reply:
i also tend to agree that we probably don't need to do random checks for ALL pages and use bank erase. 
The per page behavior and the mp region configuration is really a lot more interesting.

Reply:
2.5h wall clock time is way too long for an IP level test. Please profile the simulation by passing `--profile time` to see if there are bottlenecks that can be eliminated.

Reply:
Since we removed pointer from scoreboard,now there is waiting time between test and scoreboard for updating memory in the scoreboard. Besides that, data partition has 256 pages of 2KB size and to check if erase of entire bank works, entire bank is written,then erased and after that read.

Reply:
> 2.5h wall clock time is way too long for an IP level test. Please profile the simulation by passing `--profile time` to see if there are bottlenecks that can be eliminated.

I run profiling but there were no tasks found to consume time so I will leave it as it is. There is a knob for Bank Erase process so it is easy to control this long part of the test. First part has 100 random transactions: program, read and erase page.

Reply:
@sriyerg  @tjaychen  @eshapira : To short time of simulation, I could implement bank erase checking like in rand ops test using "cfg.flash_mem_bkdr_erase_check(flash_op, exp_data);". This would check via backdoor whether all 1s are there and it would have no need updating memory in scoreboard. For page erase, checking would remain in scoreboard.

Reply:
That will probably save some time, but I don't think it will be significant, since from my experience the time of read transaction is pretty much nothing in compare to that of erase. Did you checked its effect on the time?

Reply:
I am waiting green light from you to implement this change. It will be much quicker since there will be no time lost for preparation. First we are writing all ones to entire bank, then we program random values (dependent on the en bits), then erase entire bank and then read out. With only backdoor checking without preparation it will be much shorter. I repeat that all will remain for page erase op but from comments here, i do not see reason to do it so detailed for bank erase. It is up to you. Current test is exercising fully bank erase feature with all random en bits.